pyqg/model.py

```diff
@@ -100,6 +107,8 @@ class Model(PseudoSpectralKernel):
 
     def __init__(
         self,
+        kernel=kernels.CythonFFTWKernel,
+        kernel_kwargs={},
```

Comment:
These need to be documented in the docstring

Reply:
I've done the bare minimum to document this now (some description in the `pyqg.Model` docstring), but I'm realizing that for scalability, we probably need to add kernels to the API and give them their own documentation (which users would reference to figure out `kernel_kwargs`).